demos/apps/bookinfo/README.md

```diff
@@ -41,19 +35,30 @@ rename the tool to `istioctl`. For example:
 $ cp istioctl-osx /usr/local/bin/istioctl
 ```
 
-## Running the Bookinfo Application
+## Setup
+
+Execute the following instructions to install the Istio manager, mixer, and an envoy-based
+ingress controller, which will be used to implement the gateway for the application. 
+
+> Note: The following instructions assume your crurrent working directory is the [istio repo root](https://github.com/istio/istio).
 
-1. Bring up the Istio control plane:
+1. Follow the [Instio installation instructions](../../../kubernetes/README.md) to bring 
+   up the Istio control plane services. (Note: no addons are currently needed to run
+   the bookinfo demo.)
 
+1. Launch the ingress controller using the following command (TODO: move this to the install addons?):
+   
    ```bash
-   $ kubectl apply -f ../../istio
+   $ kubectl apply -f ./demos/ingress-controller.yaml
```

Comment:
The ingress k8s resource belongs with the bookinfo though, since it maps /productpage.

Can we use one istio ingress controller to serve, multiple ingress resources and multiple apps in multiple namespaces?